### PR TITLE
1851 tags clickable

### PR DESCRIPTION
**Pull request by Tirokk**
> [mcdead154](https://github.com/mcdead154) **Authored by [mcdead154](https://github.com/mcdead154)**
_<time datetime="2019-10-30T14:10:24Z" title="Wednesday, October 30th 2019, 3:10:24 pm +01:00">Oct 30, 2019</time>_
_Merged <time datetime="2019-11-06T08:59:12Z" title="Wednesday, November 6th 2019, 9:59:12 am +01:00">Nov 6, 2019</time>_
---

## 🍰 Pullrequest

Added feature, clickable tags below post, same as hashtags are.
### Issues

- relates #1851 

### Todo

- [X] None
image

### CODE REVIEW

File: webapp/components/Hashtag/Hashtag.vue [hunk 1]
```diff
@@ -1,5 +1,7 @@
 <template>
-  <ds-tag>#{{ id }}</ds-tag>
+  <ds-tag>
+    <nuxt-link :to="hashtagUrl">#{{ id }}</nuxt-link>
```
Comment (Tirokk) [inline webapp/components/Hashtag/Hashtag.vue:3]: > [roschaefer](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-10-31T17:06:55Z" title="Thursday, October 31st 2019, 6:06:55 pm +01:00">Oct 31, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,5 +1,7 @@
 <template>
-  <ds-tag>#{{ id }}</ds-tag>
+  <ds-tag>
+    <a v-bind:href="`/?hashtag=${id}`" target="_blank">#{{ id }}</a>
```


```suggestion
    <nuxt-link :to="hashtagUrl">#{{ id }}</nuxt-link>
```
and implement a `hashtagUrl` computed property :wink:

**Comment by Tirokk** [comment]
> [mcdead154](https://github.com/mcdead154) **Authored by [mcdead154](https://github.com/mcdead154)**
_<time datetime="2019-10-31T20:26:58Z" title="Thursday, October 31st 2019, 9:26:58 pm +01:00">Oct 31, 2019</time>_

---


@roschaefer I have tried nuxt link, but hashtag opens a new tab on click, and I haven't find a way for nuxt link to open a blank tab, so I have used an a tag instead

**Comment by Tirokk** [comment]
> [roschaefer](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-10-31T23:57:53Z" title="Friday, November 1st 2019, 12:57:53 am +01:00">Nov 1, 2019</time>_

---


`nuxt-link` does not need to open in a new tab. It's used when we link within our app. If we link to somewhere outside of our app, we would use `<a href="...">`. Among other things it does link prefetching and we would loose that optimization.

**Comment by Tirokk** [comment]
> [roschaefer](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-10-31T23:59:26Z" title="Friday, November 1st 2019, 12:59:26 am +01:00">Nov 1, 2019</time>_

---


It might be that the editor uses `<a>` tags. But that's not on purpose. It's because we did not figure out yet how to render `nuxt-link`s inside an editor node.

**Review by Mogge — APPROVED** [review]
Looks great @mcdead154 
Thanks for your contribution and making the requested changes!!

**Pull request closed without merging**